docs/index.md

```diff
@@ -25,7 +25,7 @@ Deploy Kubernetes Helm Charts
 
 March 2022 Update - The helmfile project has been moved to [helmfile/helmfile](https://github.com/helmfile/helmfile) from the former home `roboll/helmfile`. Please see roboll/helmfile#1824 for more information.
 
-Even though Helmfile is used in production environments [across multiple organizations](users.md), it is still in its early stage of development, hence versioned 0.x.
+Even though Helmfile is used in production environments [across multiple organizations](https://helmfile.readthedocs.io/en/latest/users/), it is still in its early stage of development, hence versioned 0.x.
```

Comment:
The user experience might not be smooth when the user is browsing this in the markdown view on GitHub UI or viewing locally before submitting a pull request to update this md file. That said, can we leave this as-is?
```suggestion
Even though Helmfile is used in production environments [across multiple organizations](users.md), it is still in its early stage of development, hence versioned 0.x.
```

Reply:
@mumoshu done